content/features.md

```diff
@@ -0,0 +1,50 @@
++++
+title="Features"
+weight=1
+layout="standalone"
++++
+
+
+<div class='grid'>
+  
+{{< feature title="Bill-of-Materials" >}}
+Insights into the contents of the app image.
+{{</>}}
+
+{{< feature title="Build Cache" align="right" >}}
+Robust build caching is used to improve performance.
+{{</>}}
+
+{{< feature title="Composable" >}}
+Multiple buildpacks can be used to create an app image.
+{{</>}}
+
+{{< feature title="Fast" >}}
+Only re-builds and uploads layers when necessary.
+{{</>}}
+
+{{< feature title="Multi-language" >}}
+Supports more than one programming language family.
+{{</>}}
+
+{{< feature title="Multi-process" >}}
+Image can have multiple entrypoints. Think operational modes.
```

Comment:
This wording is a bit odd. Maybe "Image can have multiple entrypoints for each operational mode."